### DISCUSSION TITLE
How do I disable git repository info next to search?

### DISCUSSION THREAD

**Discussion by csantanapr** [squidfunk/mkdocs-material#2705] [Q&A]
Hi I'm reading the docs and I can't see how to disable git repository 
https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#configuration

I tried removing or setting to empty string in mkdocs.yaml
```
repo_name: ""
repo_url: ""
```

This is what I want to remove or replace with just an icon of git no info on stars or forks.
![image](https://user-images.githubusercontent.com/1094878/119890149-7abed300-bf05-11eb-9a27-a70950b9ca24.png)

I want just the icon like this
![image](https://user-images.githubusercontent.com/1094878/119890265-9e821900-bf05-11eb-8bce-df67a9ee90f8.png)

**Comment by squidfunk** [comment]
You've two options:

1. Hide the relevant parts with CSS (simple)
2. Extend the theme (advanced)
    1. Override the [source partial](https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/source.html) as described in the customization guide
    2. Remove the `data-md-component=source` attribute, as that's used for detecting and mounting
    3. Remove the repository name and other parts of the source partial you don't need
    4. Possibly adjust spacings with CSS
